### PR TITLE
feat(drghs): updates drghs API to use new casing

### PR DESCRIPTION
**Pull request by orthros**
tldr: DRGHS's maintner API is changing its backend to gRPC in alignment with
other services in the DRGHS project. It will support a REST based API
via Cloud Endpoints HTTP=>gRPC transcoding in Extensible Service Proxy.
This commit rectifies the changes that happen in the transcoding (case
sensitivity mostly).

### CODE REVIEW

File: src/issue.ts [hunk 1]
```diff
@@ -279,12 +279,29 @@ function getTypes(i: ApiIssue) {
 }
 
 function isPullRequest(i: ApiIssue) {
-  return !!i.PullRequest;
+  return !!i.isPr;
+}
+
+export function getPriority(p: string): number {
```
Comment (JustinBeckwith) [inline src/issue.ts]: Little confused by this piece :) I would imagine we want a `p0` coming back as `0` instead of `1`. Additionally - if there's no priority on the issue, do we want a default?  Or could we make it null|undefined?

File: src/issue.ts [hunk 2]
```diff
@@ -279,12 +279,29 @@ function getTypes(i: ApiIssue) {
 }
 
 function isPullRequest(i: ApiIssue) {
-  return !!i.PullRequest;
+  return !!i.isPr;
```
Comment (JustinBeckwith) [inline src/issue.ts]: If there's an `isPr` field now, I think we can just drop this function and use that instead

File: src/issue.ts [hunk 3]
```diff
@@ -74,17 +74,17 @@ async function getRepoIssues(repo: Repo, flags?: Flags): Promise<IssueResult> {
       repo: repo.repo,
       types: getTypes(r),
       api,
-      team: getTeam(r.Repo, api),
+      team: getTeam(r.repo, api),
       isOutOfSLO: isOutOfSLO(r),
       isTriaged: isTriaged(r),
-      pri: r.PriorityUnknown ? undefined : r.Priority,
+      pri: r.priorityUnknown ? undefined : getPriority(r.priority),
```
Comment (JustinBeckwith) [inline src/issue.ts]: Given this change and the method below, can you explain exactly what changed with the API?

File: src/issue.ts (hunk 1 above)
Comment (orthros) [reply src/issue.ts]: This is so confusing, I confused myself! This should be a bit simpler after my next commit.

tldr the [new Proto spec](https://github.com/GoogleCloudPlatform/devrel-services/pull/24/files#diff-c0752684c0c2862f8d029c8e99e9e8ccR63-R70) has the 0 value as Prioroty unknown and I confused myself and wrote a translation function for that. I'll update this to be correct (i.e. P0 => 0)

File: src/issue.ts (hunk 3 above)
Comment (orthros) [reply src/issue.ts]: Sure! basically we are using a [proto enum field](https://github.com/GoogleCloudPlatform/devrel-services/pull/24/files#diff-c0752684c0c2862f8d029c8e99e9e8ccR63-R70) for the Prioroty now which the gRPC => HTTP translation serializes as a string and not as a number, necessitating the funciton

**Comment by orthros** [comment]
Okay! On the testing front, I ran a test with sloth on a subset of the repositories (since DEV and PROD don't track the same repos).

Here is the repos.json I used:

```json
{
  "repos": [
    { "language": "cpp", "repo": "googleapis/google-cloud-cpp"},

    { "language": "dotnet", "repo": "googleapis/google-cloud-dotnet"},

    { "language": "go", "repo": "googleapis/google-cloud-go"},

    { "language": "nodejs", "repo": "googleapis/nodejs-spanner", "apiHint": "spanner"},

    { "language": "php", "repo": "googleapis/google-cloud-php"},
    { "language": "php", "repo": "GoogleCloudPlatform/php-docs-samples"},

    { "language": "python", "repo": "googleapis/google-cloud-python"},
    { "language": "python", "repo": "GoogleCloudPlatform/python-docs-samples"},

    { "language": "ruby", "repo": "googleapis/google-cloud-ruby"}
  ]
}
```

I then ran `sloth issues > out.txt` with this PR and with `master`  and diffed the two. Here are the results:

```diff
336c336
< │ https://github.com/googleapis/google-cloud-dotnet/issues/2163          │ 🦖      │ 🦖    │ Create a Timed and Sized buffer combination for the Google.Cloud.Diagnostic… │ dotnet   │ clouderrorreporting  │ 2   │ core           │
---
> │ https://github.com/googleapis/google-cloud-dotnet/issues/2163          │ 🦖      │ 🦖    │ Create a Timed and Sized buffer combination for the Google.Cloud.Diagnostic… │ dotnet   │ cloudtrace           │ 2   │ core           │
376,377d375
< │ https://github.com/googleapis/google-cloud-go/issues/361               │ 🦖      │ 🦖    │ compute: provide easier way to add container optimized OS options for deplo… │ go       │ datastore            │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
534,535d531
< │ https://github.com/googleapis/google-cloud-go/issues/1709              │ 🚨      │ 🚨    │ Cannot deploy Cloud Run using API: badRequest INVALID_ARGUMENT (no further … │ go       │                      │     │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
842,853d837
< │ https://github.com/googleapis/google-cloud-python/issues/5             │ 🦖      │ 🦖    │ NDB: Enable a default model                                                  │ python   │ storage              │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/7             │ 🦖      │ 🦖    │ Add `tzinfo` to `DatetimeProperty`                                           │ python   │                      │     │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/8             │ 🦖      │ 🦖    │ Figure out discrepancy between Google Cloud SDK and `datastore-ndb-python`   │ python   │ datastore            │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/9             │ 🦖      │ 🦖    │ Ditch sphinx-docstring-typing / use type annotations for `ndb`               │ python   │                      │     │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/16            │ 🦖      │ 🦖    │ Add support for remapping `ndb` module paths when unpickling                 │ python   │ datastore            │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/18            │ 🦖      │ 🦖    │ Datastore: speed regression on py37 v.s. GAE NDB on py27                     │ python   │ storage              │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
874c858
< │ https://github.com/googleapis/google-cloud-python/issues/4840          │ 🦖      │ 🦖    │ Determine how to support emulators consistently across all clients.          │ python   │ storage              │     │ soda           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/4840          │ 🦖      │ 🦖    │ Determine how to support emulators consistently across all clients.          │ python   │ bigtable             │     │ soda           │
892c876
< │ https://github.com/googleapis/google-cloud-python/issues/5448          │ 🦖      │ 🦖    │ No documentation for many auto generated types.                              │ python   │ redis                │     │ soda           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/5448          │ 🦖      │ 🦖    │ No documentation for many auto generated types.                              │ python   │ cloudasset           │     │ core           │
896c880
< │ https://github.com/googleapis/google-cloud-python/issues/5520          │ 🦖      │ 🦖    │ Regen / release vision, texttospeech, iot                                    │ python   │ texttospeech         │     │ animl          │
---
> │ https://github.com/googleapis/google-cloud-python/issues/5520          │ 🦖      │ 🦖    │ Regen / release vision, texttospeech, iot                                    │ python   │ vision               │     │ animl          │
924c908
< │ https://github.com/googleapis/google-cloud-python/issues/6234          │ 🦖      │ 🦖    │ Generated noxfiles missing 'cover' session                                   │ python   │ automl               │     │ animl          │
---
> │ https://github.com/googleapis/google-cloud-python/issues/6234          │ 🦖      │ 🦖    │ Generated noxfiles missing 'cover' session                                   │ python   │ dataproc             │     │ animl          │
958c942
< │ https://github.com/googleapis/google-cloud-python/issues/7291          │ 🦖      │ 🦖    │ Document using JSON string to create credentials from service account.       │ python   │ texttospeech         │     │ animl          │
---
> │ https://github.com/googleapis/google-cloud-python/issues/7291          │ 🦖      │ 🦖    │ Document using JSON string to create credentials from service account.       │ python   │ cloudtrace           │     │ core           │
964c948
< │ https://github.com/googleapis/google-cloud-python/issues/7503          │ 🦖      │ 🦖    │ Missing documentation for 'enum' modules                                     │ python   │ bigquerystorage      │     │ core           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/7503          │ 🦖      │ 🦖    │ Missing documentation for 'enum' modules                                     │ python   │ scheduler            │     │ core           │
1002c986
< │ https://github.com/googleapis/google-cloud-python/issues/8174          │ 🦖      │ 🦖    │ Autogen: fix coverage landmines.                                             │ python   │ webrisk              │     │ core           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/8174          │ 🦖      │ 🦖    │ Autogen: fix coverage landmines.                                             │ python   │ vision               │     │ animl          │
1078c1062
< │ https://github.com/googleapis/google-cloud-python/issues/9298          │ 🦖      │ 🦖    │ Add Retry Strategy to manual libraries.                                      │ python   │ logging              │     │ core           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/9298          │ 🦖      │ 🦖    │ Add Retry Strategy to manual libraries.                                      │ python   │ bigtable             │     │ soda           │
1084c1068
< │ https://github.com/googleapis/google-cloud-python/issues/9457          │ 🦖      │ 🦖    │ [BigQuery Data Transfer Service, all GAPIC clients]: provide a close() meth… │ python   │ core                 │     │ core           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/9457          │ 🦖      │ 🦖    │ [BigQuery Data Transfer Service, all GAPIC clients]: provide a close() meth… │ python   │ bigquerydatatransfer │     │ animl          │
1106c1090
< │ https://github.com/googleapis/google-cloud-python/issues/9552          │ 🦖      │ 🦖    │ Storage, Core: Once x-goog-api-client headers, gccl, is used for storage, r… │ python   │ storage              │     │ soda           │
---
> │ https://github.com/googleapis/google-cloud-python/issues/9552          │ 🦖      │ 🦖    │ Storage, Core: Once x-goog-api-client headers, gccl, is used for storage, r… │ python   │ core                 │     │ core           │
1110c1094
< │ https://github.com/googleapis/google-cloud-python/issues/9580          │ 🦖      │ 🦖    │ Testing: rationalize / normalize VPCSC environment detection in systests     │ python   │ dlp                  │     │ animl          │
---
> │ https://github.com/googleapis/google-cloud-python/issues/9580          │ 🦖      │ 🦖    │ Testing: rationalize / normalize VPCSC environment detection in systests     │ python   │ monitoring           │     │ core           │
1228,1229d1211
< │ https://github.com/googleapis/google-cloud-python/issues/9990          │ 🚨      │ 🚨    │ BigQuery: 400 PUT: Unknown upload notification type: 5                       │ python   │ bigquery             │     │ animl          │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
1612,1613d1593
< │ https://github.com/googleapis/google-cloud-ruby/issues/759             │ 🦖      │ 🦖    │ Streaming Google storage upload                                              │ ruby     │                      │ 2   │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
1618c1598
< │ https://github.com/googleapis/google-cloud-ruby/issues/2142            │ 🦖      │ 🦖    │ Stackdriver is not working on my Rails project                               │ ruby     │ clouddebugger        │     │ core           │
---
> │ https://github.com/googleapis/google-cloud-ruby/issues/2142            │ 🦖      │ 🦖    │ Stackdriver is not working on my Rails project                               │ ruby     │ clouderrorreporting  │     │ core           │
```

A quick summary of the results:

There are some issues that are in DEV that are not in PROD, this is expected as they are in the `google-cloud-python` repository, as there was an intermittet issue in maniter that just never picked up those issues.

The other differences you will see are in the `API` column.  I have yet to dig too deeply but it appears as though in cases where there are multiple labels on an issue that correspond to an "api" label, it returns the first label available.

e.g. For issue https://github.com/googleapis/google-cloud-ruby/issues/2142, it has API labels
```
api: clouddebugger
api: clouderrorreporting
api: cloudtrace
api: logging
```
And sloth returns different areas for each one.

In both DEV and PROD maintner, the `Label` array does not return Labels in a stable manner which makes this "expected"

I added a `.sort()` to the `getAPI` function and this is the new diff

```diff
376,377d375
< │ https://github.com/googleapis/google-cloud-go/issues/361               │ 🦖      │ 🦖    │ compute: provide easier way to add container optimized OS options for deplo… │ go       │ datastore            │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
534,535d531
< │ https://github.com/googleapis/google-cloud-go/issues/1709              │ 🚨      │ 🚨    │ Cannot deploy Cloud Run using API: badRequest INVALID_ARGUMENT (no further … │ go       │                      │     │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
842,853d837
< │ https://github.com/googleapis/google-cloud-python/issues/5             │ 🦖      │ 🦖    │ NDB: Enable a default model                                                  │ python   │ storage              │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/7             │ 🦖      │ 🦖    │ Add `tzinfo` to `DatetimeProperty`                                           │ python   │                      │     │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/8             │ 🦖      │ 🦖    │ Figure out discrepancy between Google Cloud SDK and `datastore-ndb-python`   │ python   │ datastore            │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/9             │ 🦖      │ 🦖    │ Ditch sphinx-docstring-typing / use type annotations for `ndb`               │ python   │                      │     │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/16            │ 🦖      │ 🦖    │ Add support for remapping `ndb` module paths when unpickling                 │ python   │ datastore            │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
< │ https://github.com/googleapis/google-cloud-python/issues/18            │ 🦖      │ 🦖    │ Datastore: speed regression on py37 v.s. GAE NDB on py27                     │ python   │ storage              │     │ soda           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
1228,1229d1211
< │ https://github.com/googleapis/google-cloud-python/issues/9990          │ 🚨      │ 🚨    │ BigQuery: 400 PUT: Unknown upload notification type: 5                       │ python   │ bigquery             │     │ animl          │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
1614,1615d1595
< │ https://github.com/googleapis/google-cloud-ruby/issues/759             │ 🦖      │ 🦖    │ Streaming Google storage upload                                              │ ruby     │                      │ 2   │ core           │
< ├────────────────────────────────────────────────────────────────────────┼─────────┼───────┼──────────────────────────────────────────────────────────────────────────────┼──────────┼──────────────────────┼─────┼────────────────┤
```

Which all correspond to the "missing" issues covered earlier. 

With this in mind, do you want me to add the `.sort()` declaration to both the `getAPI` and `getTypes` function?

File: src/issue.ts (hunk 3 above)
Comment (JustinBeckwith) [reply src/issue.ts]: Rad.  Just please make sure something like this ends up in a comment in that function :)

File: src/issue.ts [hunk 4]
```diff
@@ -278,13 +278,26 @@ function getTypes(i: ApiIssue) {
   return types;
 }
 
-function isPullRequest(i: ApiIssue) {
-  return !!i.PullRequest;
+export function getPriority(p: string): number | undefined {
```
Comment (JustinBeckwith) [inline src/issue.ts]: Or....

```js
const getPriority = (p: string) => Number(p.toLowerCase().slice(1));
```

No need for any of this I say!

**Comment by JustinBeckwith** [comment]
On the sort - yeah that's fine.  If you include multiple APIs ... not sure what you would want :)

File: src/issue.ts (hunk 4 above)
Comment (orthros) [reply src/issue.ts]: ack
